kubernetes-workshop/8004_network/modul_network_8004_01_slides.md

```diff
@@ -77,9 +90,9 @@ Two types of services allow external access
 ## LoadBalancer
 
 * Requests an external loadbalancer
-* Mostly relevant for cloud environments
+* Mostly relevant for cloud environments and non trivial (ie. production) on-prem deploys
```

Comment:
I have the feeling you're missing a for here?

```suggestion
* Mostly relevant for cloud environments and non trivial for (ie. production) on-prem deploys
```

Reply:
It's for non trivial on-prem deploys, it's not non trivial.

Reply:
Didn't know that trivial could also have that meaning. English is a strange language.